__tests__/beers.test.js

```diff
@@ -0,0 +1,75 @@
+import pool from '../lib/utils/pool.js';
+import setup from '../data/setup.js';
+import request from 'supertest';
+import app from '../lib/app.js';
+import Beer from '../lib/models/Beer.js';
+
+
+describe('beer routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('creates a beer with POST', async () => {
+    const pilsnerUrquell = { beerName: 'pilsner urquell', nationality: 'czech republic', variety: 'pilsner', alcoholPercentage: 4 };
+
+    const res = await request(app)
+      .post('/api/v1/beers')
+      .send(pilsnerUrquell);
+    
+    expect(res.body).toEqual({
+      id: '1',
+      ...pilsnerUrquell
+    });
+  });
+
+  it('gets a beer by id with GET', async () => {
+    const pilsnerUrquell = await Beer.insert({ beerName: 'pilsner urquell', nationality: 'czech republic', variety: 'pilsner', alcoholPercentage: 4 });
+
+    const res = await request(app)
+      .get(`/api/v1/beers/${pilsnerUrquell.id}`);
+      
+    
+    expect(res.body).toEqual(pilsnerUrquell);
+  });
+
+  it('gets all beers with GET', async () => {
+    const pilsnerUrquell = await Beer.insert({ beerName: 'pilsner urquell', nationality: 'czech republic', variety: 'pilsner', alcoholPercentage: 4 });
```

Comment:
Great work, Ben! Looks great!

I did want to mention, to make tests a little bit drier, you can add a beforeAll in a test, and create a beer. So that it would be, before all tests, a beer is inserted. That way, you don't have to insert one for each test. Not necessary for this, but thought it could be a nice suggestion for the future.